### PR TITLE
Send body regardless of what the HTTP method is.

### PR DESCRIPTION
**Pull request by keichan34**
This allows PUT and PATCH requests, for example. This could be also accomplished by using a whitelist of HTTP methods where the request body should also be forwarded, but I think that's a little more error-prone than just sending everything. If it's really a bad request, it should be filtered by API Gateway or ALB anyways.

(I also added `-e` to the build script, because it was failing midway but still continuing. I see that the specific error I was encountering has already been fixed in master, is there a chance we can merge master back to development soon?)

### CODE REVIEW

File: bootstrap [hunk 1]
```diff
@@ -171,13 +171,7 @@ while (true) {
...
 
   if (strlen($body) > 0) {
-    if($event['httpMethod'] === 'POST'){
-      curl_setopt($ch, CURLOPT_POSTFIELDS, $body);
-    }    
-    curl_setopt($ch, CURLOPT_INFILESIZE, strlen($body));
-    curl_setopt($ch, CURLOPT_READFUNCTION, function ($ch, $fd, $length) use ($body) {
-      return $body;
-    });
+    curl_setopt($ch, CURLOPT_POSTFIELDS, $body);
```
Comment (txase) [inline bootstrap:174]: Why do we no longer need `CURLOPT_INFILESIZE` and `CURLOPT_READFUNCTION`?

**Review by txase — COMMENTED** [review]
:pray:

File: bootstrap (hunk 1 above)
Comment (keichan34) [reply bootstrap:174]: The `CURLOPT_READFUNCTION` callback is only called if there was a file stream specified in `CURLOPT_INFILE`, so it wasn't being called in this instance.

**Review by txase — APPROVED** [review]
:pray:

**Pull request merged**